packages/ckeditor5-engine/src/conversion/downcasthelpers.js

```diff
@@ -950,22 +1017,25 @@ export function wrap( elementCreator ) {
  * @param {Function} elementCreator Function returning a view element, which will be inserted.
  * @returns {Function} Insert element event converter.
  */
-export function insertElement( elementCreator ) {
+export function insertElement( elementCreator, consumer ) {
```

Comment:
JSDoc added. For the default value, I absolutely agree. It would make no sense to keep it without default so every time you want to use `insertElement` you have to specify the consumer function that just consumes the element and nothing else (default back-compat behaviour).